submatrix/homeserver.yaml.template

```diff
@@ -0,0 +1,37 @@
+server_name: "${MATRIX_HOSTNAME}"
+pid_file: /homeserver.pid
+listeners:
+  - port: 8008
+    tls: false
+    type: http
+    bind_addresses: ['::1', '127.0.0.1']
+    resources:
+      - names:
+        - client
+        compress: false
```

Comment:
Since we dont need register_new_matrix_user anymore do we even need this listener anymore?